### PR TITLE
anticipate and strengthen FOSS recommendation

### PR DESCRIPTION
**Pull request by zacchiro**
concrete wording proposal, as per #1

### CODE REVIEW

File: faq.md [hunk 1]
```diff
@@ -59,13 +60,14 @@ software.
...
-that software is Free Software.
+that software is [Free Software](https://www.gnu.org/philosophy/free-sw.html).
 
 ## Which license should I choose? {#which-license}
 
-Ultimately, the license you choose is up to you. If you are contributing to an
-existing project, you should release your changes under the same license as the
-project. Otherwise, the [Free Software
+Always choose a Free Software license, i.e., a license that gives the recipient
+the four freedoms mentioned above. Aside from that, the license you choose is up
```
Comment (carmenbianca) [inline faq.md]: "mentioned above" refers to something found on the linked website, which may be a bit confusing for people who don't click on every link. I would probably use the wording "... a license that gives the recipient the freedom to use, study, share, and improve the software".

**Comment by carmenbianca** [comment]
Looks good to me!

@mxmehl

File: faq.md [hunk 2]
```diff
@@ -13,9 +13,10 @@ propose three steps to achieve this:
 
 ### 1. Choose and provide licenses {#step-1}
 
-Find the SPDX identifier of your license in the [SPDX License
-List](https://spdx.org/licenses/). Download the license text for your license
-from the
+Choose a [Free Software license](https://www.gnu.org/licenses/licenses.html)
```
Comment (mxmehl) [inline faq.md]: I am uncertain about the link to gnu.org. Is it to explain the term *Free Software license*, or shall it help picking a license?

If the former, I find it a bit too specific for people woh don't know much about software licenses.
If the latter, we should link to `#which-license` where also invite people to use other services for recommendations.

File: faq.md [hunk 3]
```diff
@@ -59,13 +60,15 @@ software.
 
 If the license allows the recipient to [use, study, share and
 improve](https://fsfe.org/freesoftware/basics/4freedoms.html) the software, then
-that software is Free Software.
+that software is [Free Software](https://www.gnu.org/philosophy/free-sw.html).
```
Comment (mxmehl) [inline faq.md]: Please change the link to `https://fsfe.org/about/basics/freesoftware` :)

**Comment by mxmehl** [comment]
Great, thank you @zacchiro!

**Comment by silverhook** [comment]
Perfect, thank you @zacchiro !

**Pull request merged**